modules/oe_showcase_list_pages/tests/src/ExistingSite/ListPagesTest.php

```diff
@@ -223,6 +230,26 @@ public function testCreateListPages() {
     $title_input->setValue('News number 1');
     $search_button->click();
     $this->assertSearchResultsTitle('Event list page', 0);
+
+    // Filter results by location.
+    $filter_form->findButton('Clear filters')->click();
+    $location = $filter_form->findField('Location');
+    $location->selectOption('FR');
```

Comment:
We should verify that country names are spelled out.
```suggestion
    $location->selectOption('France');
```